pkg/interactive/validation.go

```diff
@@ -65,6 +69,9 @@ func IsCert(filepath interface{}) error {
 		if s == "" {
 			return nil
 		}
+		if s == ocm.DoubleQuotesToRemove {
```

Comment:
This creates an unnecessary dependency on the `ocm` package. I think this `const` should live inside this package instead so that the import above can be removed.

Reply:
defined it locally.